src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Shapes/Shapes_Tests.cs

```diff
@@ -20,14 +20,14 @@ public void Draw_polyline()
 		{
 			Run("SamplesApp.Windows_UI_Xaml_Shapes.PolylinePage");
 			_app.WaitForElement("DPolyline");
-			TakeScreenshot($"PolylinePage");
+			TakeScreenshot($"PolylinePage", ignoreInSnapshotCompare: true);
```

Comment:
Those tests rely on the result of the screenshots comparison for validation. What makes them fail ?